src/examples/src/widgets/header/Basic.tsx

```diff
@@ -0,0 +1,20 @@
+import Header from '@dojo/widgets/header';
+import { Link } from '@dojo/framework/routing/Link';
+import { create, tsx } from '@dojo/framework/core/vdom';
+
+const factory = create();
+
+export default factory(function Basic() {
+	return (
+		<Header>
+			{{
+				actionsRenderer: () => [
```

Comment:
i understand that these child renderers are alphabetical, but do you think you could do them in the order they're displayed instead as it aligns better between the render and the example.